scitsr/data/loader.py

```diff
@@ -107,7 +107,7 @@ def load_dataset(self, dataset_dir, with_cells, trim=None, debug=False, exts=Non
         dataset, cells = [], []
         if exts is None: exts = ['chunk', 'rel']
         if with_cells:
-            exts.append('json')
+            exts.append('structure')
```

Comment:
At line 198: `sub_path = os.path.join(root_dir, ext, name + '.' + ext)` So the files in the 'structure' folder should have the suffix of 'structure'.

I am not sure whether it is okay to directly update this line without updating line 198. Of course, we can also rename the files in this folder with the suffix of 'structure'.

Reply:
You are correct. I missed it. Let the user take care by renaming the folder